app/views/balances/index.html.erb

```diff
@@ -0,0 +1,32 @@
+<section class="transaction">
+  <div class="nav flex">
+    <svg xmlns="http://www.w3.org/2000/svg" width="50" height="50" viewBox="0 0 24 24"><path fill="currentColor" d="M4 18h16c.55 0 1-.45 1-1s-.45-1-1-1H4c-.55 0-1 .45-1 1s.45 1 1 1zm0-5h16c.55 0 1-.45 1-1s-.45-1-1-1H4c-.55 0-1 .45-1 1s.45 1 1 1zM3 7c0 .55.45 1 1 1h16c.55 0 1-.45 1-1s-.45-1-1-1H4c-.55 0-1 .45-1 1z"/></svg>
+    <h1>TRANSACTIONS</h1>
+    <svg xmlns="http://www.w3.org/2000/svg" width="50" height="50" viewBox="0 0 24 24"><path fill="none" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M3 10a7 7 0 1 0 14 0a7 7 0 1 0-14 0m18 11l-6-6"/></svg>
+  </div>
+
+  <div class="items">
+  <ul>
+    <% @balances.each do |balance| %>
+      <%= link_to "/balances/#{balance.id}" do %>
+        <li>
+          <img src=<%= balance.icon %> alt=<%= balance.name %> >
+          <div class="balance-info">
+            <p class="balance-name"><%= balance.name.capitalize %></p>
+            <p class="created-at"><%= balance.created_at.strftime('%d-%b-%Y') %></p>
+          </div>
+          <p class="total-amount">$<%= balance.total_amount %></p>
+        </li>
+      <% end %>
+    <% end %>
+  </ul>
+</div>
+
+  <div class="scan-link flex">
+    <span class="flex">
+      <%= link_to 'SCAN', new_balance_path, class: "span flex" %>
```

Comment:
- [ ] As per the requirement, please change this button name to 'add a new category'.